evaluator/images/cargo/entry.py

```diff
@@ -195,11 +195,18 @@ def run_cargo(command: str, args: List[str]) -> BuildResult:
 
 COMMAND = get_param("cmd", "build")
 ARGS = get_param("args", [], parse_json=True)
+SANITIZED_FILES = ["result.html", "piperesult.json"]
 
 try:
     result = run_cargo(COMMAND, ARGS)
 except BaseException as e:
     result = BuildResult.fail(f"Cargo execution failed\n{e}")
+finally:
+    for file in SANITIZED_FILES:
+        try:
+            os.unlink(file)
+        except FileNotFoundError:
```

Comment:
Let's use `# noqa` here, same as in the `gcc` pipeline, and catch everything, to make sure that all files are unlinked.